### ISSUE TITLE
New WCAG Criteria - 3.2.6 Consistent Help  - A

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2943]
For onrr.gov, we do a good job of linking to contacts, but all the FAQs are in places where the user already needs to have made it to that part of the site. We should consider adding page descriptions for the FAQs that include words like Help, FAQs, etc so these come up in site search results.

**Comment by Maroyafaied** [comment]
add meta data descriptions to the trouble shooting pages.

**Comment by AlexandracmONRR** [comment]
Alex's group to update:

- 1.3.2 System access troubleshooting
- 2.1.5 oil and gas production troubleshooting
- 2.2.5 oil and gas revenue troubleshooting
- 2.3.5 solid minerals troubleshooting
- 2.4.5 geothermal troubleshooting
- 7.9.2.2 System Access (troubleshooting section)
- 7.9.2.3 Lease & Agreement (troubleshooting section)
- 7.9.2.4 Oil & Gas Production (troubleshooting section)
- 7.9.2.5 Oil & Gas Royalty (troubleshooting section)
- 7.9.2.6 Oil & Gas Rent (troubleshooting section)
- 7.9.2.7 Solid Minerals (troubleshooting section)
- 7.9.2.8 Geothermal (troubleshooting section)


Erin's group to update:

- 4.4.1 contacts troubleshooting
- 7.9.2.3 Lease & Agreement (troubleshooting section)
- 7.9.2.1 Getting Started (troubleshooting section)
- 7.9.2.9 Payments & Debt Collection (troubleshooting section)
- 7.9.2.10 Updating Contact Info (troubleshooting section)
- 7.9.3.1 Valuation & Pricing (troubleshooting section)
- 7.9.3.2 Compliance & Enforcement (troubleshooting section)
- 7.9.3.3 Indian Services (troubleshooting section)
- 7.9.3.4 Information Request (troubleshooting section)

**Comment by eelzi-ONRR** [comment]
@AlexandracmONRR I reviewed search terms for CY23 and the following were used that we should make sure are added to each of these meta descriptions: help, faq, customer service. Here's an example I wrote for https://onrr.gov/about/contact/lease-agreement-contacts?
“Help with Lease & Agreements questions: specialist customer service contacts & FAQs.”

**Comment by eelzi-ONRR** [comment]
Added metadescriptions for: 
- 4.4.1 contacts troubleshooting
- 7.9.2.3 Lease & Agreement (troubleshooting section)
- 7.9.2.1 Getting Started (troubleshooting section)
- 7.9.2.9 Payments & Debt Collection (troubleshooting section)
- 7.9.2.10 Updating Contact Info (troubleshooting section)

The rest of my list doesn't have a troubleshooting section on that page. But I did add a description for the Indian Services contacts tabs.

**Comment by AlexandracmONRR** [comment]
I added a meta data description for my sections: 

- 1.3.2 System access troubleshooting
- 2.1.5 oil and gas production troubleshooting
- 2.2.5 oil and gas revenue troubleshooting
- 2.3.5 solid minerals troubleshooting
- 2.4.5 geothermal troubleshooting
- 7.9.2.2 System Access (troubleshooting section)
- 7.9.2.3 Lease & Agreement (troubleshooting section)
- 7.9.2.4 Oil & Gas Production (troubleshooting section)
- 7.9.2.5 Oil & Gas Royalty (troubleshooting section)
- 7.9.2.6 Oil & Gas Rent (troubleshooting section)
- 7.9.2.7 Solid Minerals (troubleshooting section)
- 7.9.2.8 Geothermal (troubleshooting section)

In some cases, I added onto what was already written for the metadata description.
